manim/camera/camera.py

```diff
@@ -36,42 +36,38 @@ class Camera:
 
     This is the object which takes care of what exactly is displayed
     on screen at any given moment.
-
-    Some important configuration values and local variables to note are:
-
-    background_image : :class:`str`, optional
-        The path to an image that should be the background image.
-        If not set, the background is filled with `self.background_color`
-
-    pixel_height : :class:`int`, optional
-        The height of the scene in pixels.
-
     """
 
     def __init__(
         self,
-        background_image=None,
-        frame_center=ORIGIN,
-        image_mode="RGBA",
-        n_channels=4,
-        pixel_array_dtype="uint8",
-        z_buff_func=lambda m: np.round(m.get_center()[2], 2),
-        cairo_line_width_multiple=0.01,
-        use_z_index=True,
-        background=None,
-        pixel_height=None,
-        pixel_width=None,
-        frame_height=None,
-        frame_width=None,
-        frame_rate=None,
+        background_image: str | None = None,
+        frame_center: np.ndarray = ORIGIN,
+        image_mode: str = "RGBA",
+        n_channels: int = 4,
+        pixel_array_dtype: str = "uint8",
+        cairo_line_width_multiple: float = 0.01,
+        use_z_index: bool = True,
+        background: np.ndarray | None = None,
+        pixel_height: int | None = None,
+        pixel_width: int | None = None,
+        frame_height: float | None = None,
+        frame_width: float | None = None,
+        frame_rate: float | None = None,
         **kwargs,
     ):
         """Initialises the Camera.
 
         Parameters
         ----------
-        background : optional
+        background_image
+            The path to an image that should be the background image.
+            If not set, the background is filled with `self.background_color`
```

Comment:
```suggestion
            If not set, the background is filled with :attr:`background_color`
```

or

```suggestion
            If not set, the background is filled with ``self.background_color``
```

(I'd prefer the first.)